src/core/lombok/Version.java

```diff
@@ -0,0 +1,26 @@
+package lombok;
+
+/**
+ * @See VersionableUtils
+ * @author Al3x G0m3z
+ *
+ */
+public enum Version
+{
+    v1,
+    v2,
+    v3,
+    v4,
+    v5,
+    v6,
+    v7,
+    v8,
+    v9,
+    v10,
+    v11,
+    v12,
+    v13,
+    v14,
+    v15,
+    v16
+}

```

Comment:
I'd actually propose a String for the annotation (with int for the version number). Imagine a property present in versions 3 to 9, tentatively removed in version 10, put back temporarily in version 11 and finally removed in version 16. This sounds terribly complicated, but it's as simple as
`@Versionable(versions="3-9, 11-16")`
when using strings. With many versions, doing anything else is verbose and error-prone. For a property introduced in version 6, I'd use
`@Versionable(versions="6-")`
which means you don't have to touch the annotation as long as you keep the property.